Assets/UI/Panels/citypaneloverview.lua

```diff
@@ -675,11 +725,14 @@ function ViewPanelHousing( data:table )
     Controls.HousingPopulationStatus:SetColorByName( colorName );
 
     Controls.CitizensNum:SetText( data.Population );
-    if data.Population <= 1 then
-        Controls.CitizensName:SetText(Locale.Lookup("LOC_HUD_CITY_CITIZEN"));
-    elseif data.Population > 1 then
-        Controls.CitizensName:SetText(Locale.Lookup("LOC_HUD_CITY_CITIZENS"));
-    end
+
+    Controls.CitizensName:SetText(Locale.Lookup("LOC_HUD_CITY_CITIZENS_LABEL", data.Population));
```

Comment:
the LOC_HUD_CITY_CITIZENS_LABEL is used in the unmodified Firaxis version of this file and automatically handles the pluralization of the word "Citizen" when there is more than 1 citizen in the city.